### PR TITLE
Add definitions for plur

### PR DESCRIPTION
**Pull request by iRoachie**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by iRoachie** [comment]
Closed in favour of https://github.com/sindresorhus/plur/pull/19

File: types/plur/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Type definitions for plur 3.0
+// Project: https://github.com/sindresorhus/plur#readme
+// Definitions by: Kyle Roach <https://github.com/iRoachie>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default function plur(
```
Comment (minestarks) [inline types/plur/index.d.ts]: The correct way to export this, based on the implementation, would be

```ts
function plur( 
    word: string,
    plural: string | number,
    count?: number ): string
export = plur;
```

File: types/plur/plur-tests.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import plur from 'plur';
```
Comment (minestarks) [inline types/plur/plur-tests.ts]: The correct way to import this, based on the implementation, would be:

`import plur = require('plur')`

An explanation can be found in the readme: https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

> For an NPM package, `export =` is accurate if `node -p 'require("foo")'` is the export, and `export default` is accurate if `node -p 'require("foo").default'` is the export.

**Review by minestarks — CHANGES_REQUESTED** [review]

File: types/plur/index.d.ts (hunk 1 above)
Comment (iRoachie) [reply types/plur/index.d.ts]: That fails from tsc. A 'declare' modifier is required for a top level declaration in a .d.ts file.

Comment (iRoachie) [reply types/plur/index.d.ts]: screen shot 2018-09-07 at 8 07 49 pm

Comment (minestarks) [reply types/plur/index.d.ts]: Yes, youll want to add `declare` in the beginning.

**Review by minestarks — APPROVED** [review]

**Pull request merged**